### PR TITLE
Update UG command summary

### PR DESCRIPTION
**Pull request by alvintan01**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -906,17 +910,17 @@ see `stock.txt, order.txt, prescription.txt` in that folder.
 Command | Command Syntax
 ------ | ------
 addstock | `addstock n/NAME p/PRICE q/QUANTITY e/EXPIRY_DATE {d/DESCRIPTION m/MAX_QUANTITY}`
-deletestock | `deletestock [i/STOCK_ID expiring/DATE]`
-updatestock | `updatestock i/STOCK_ID [n/NAME p/PRICE q/QUANTITY e/EXPIRY_DATE d/DESCRIPTION m/MAX_QUANTITY]`
-liststock | `liststock {i/STOCK_ID p/PRICE q/QUANTITY e/EXPIRY_DATE d/DESCRIPTION m/MAX_QUANTITY sort/COLUMN_NAME rsort/COLUMN NAME}`
+deletestock | `deletestock [i/ID expiring/DATE]`
+updatestock | `updatestock i/ID [n/NAME p/PRICE q/QUANTITY e/EXPIRY_DATE d/DESCRIPTION m/MAX_QUANTITY]`
+liststock | `liststock {i/ID p/PRICE q/QUANTITY low/LESS_THAN_OR_EQUAL_QUANTITY e/EXPIRY_DATE expiring/LESS_THAN_OR_EQUAL_EXPIRY_DATE d/DESCRIPTION m/MAX_QUANTITY sort/COLUMN_NAME rsort/COLUMN NAME}`
```
Comment (a-tph) [inline docs/UserGuide.md:915]: Should be `COLUMN_NAME` for rsort instead?

File: docs/UserGuide.md [hunk 2]
```diff
@@ -204,8 +204,11 @@ Lists all existing medications in the inventory.
 * All parameters for `liststock` command are optional, you can choose to list medication by any of the parameters.
 * You are able to `liststock` by any column and sort or reverse sort them.
 * When you update an order information, MediVault reflects the pending stocks shown here.
+* The total pending quantity will be shown if there are orders for a medication.
+* `low/LESS_THAN_OR_EQUAL_QUANTITY` and `expiring/LESS_THAN_OR_EQUAL_EXPIRY_DATE` parameters can be used to search for
+stocks with low quantity and expiring stocks. 
 
-Format: `liststock {i/STOCK_ID n/NAME p/PRICE q/QUANTITY e/EXPIRY_DATE d/DESCRIPTION m/MAX_QUANTITY sort/COLUMN_NAME rsort/COLUMN_NAME}`
+Format: `liststock {i/ID p/PRICE q/QUANTITY low/LESS_THAN_OR_EQUAL_QUANTITY e/EXPIRY_DATE expiring/LESS_THAN_OR_EQUAL_EXPIRY_DATE d/DESCRIPTION m/MAX_QUANTITY sort/COLUMN_NAME rsort/COLUMN NAME}`
```
Comment (a-tph) [inline docs/UserGuide.md:211]: Should be `COLUMN_NAME` for rsort instead?

File: docs/UserGuide.md (hunk 1 above)
Comment (alvintan01) [reply docs/UserGuide.md:915]: Noted, will update it

File: docs/UserGuide.md (hunk 2 above)
Comment (alvintan01) [reply docs/UserGuide.md:211]: Noted, will update it

**Review by a-tph — COMMENTED** [review]
LGTM

**Pull request merged**